library/core/src/num/mod.rs

```diff
@@ -1581,3 +1588,119 @@ from_str_radix_size_impl! { i16 isize, u16 usize }
 from_str_radix_size_impl! { i32 isize, u32 usize }
 #[cfg(target_pointer_width = "64")]
 from_str_radix_size_impl! { i64 isize, u64 usize }
+
+#[unstable(feature = "kani", issue = "none")]
+mod verify {
+    use super::*;
+
+    // `unchecked_add` proofs
+    //
+    // Target types:
+    // i{8,16,32,64,128} and u{8,16,32,64,128} -- 10 types in total
```

Comment:
Can you also include `isize` and `usize`?

Reply:
For sure. I'll add it.